### PR TITLE
LPD-8488 / LPD-8489 / LPD-8490 / LPD-8491 / LPD-8492 Remove unused @Reference variables - BPM components

### PR DESCRIPTION
**Pull request by diogomelo09**
[LPD-8488](https://liferay.atlassian.net/browse/LPD-8488) [LPD-8489](https://liferay.atlassian.net/browse/LPD-8489) [LPD-8490](https://liferay.atlassian.net/browse/LPD-8490) [LPD-8491](https://liferay.atlassian.net/browse/LPD-8491) [LPD-8492](https://liferay.atlassian.net/browse/LPD-8492)

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:workflow

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
Hey @pedro-oliveira446 , if these test suites pass, please also run `ci:test:kaleo-forms` and  `ci:test:workflow-metrics`.

**Comment by pedro-oliveira446** [comment]
Hey @diogomelo09,
can you run 
`LocalFile.KaleoFormsNewProcess#NewProcessAppearsOnProcessBuilder` locally? This failure seems to be related to the PR.

File: modules/dxp/apps/portal-workflow-kaleo-forms/portal-workflow-kaleo-forms-web/src/main/java/com/liferay/portal/workflow/kaleo/forms/web/internal/asset/KaleoProcessAssetRendererFactory.java [hunk 1]
```diff
@@ -113,11 +111,6 @@ public boolean hasPermission(
 	@Reference
 	private KaleoProcessLocalService _kaleoProcessLocalService;
 
-	@Reference(
-		target = "(javax.portlet.name=" + KaleoFormsPortletKeys.KALEO_FORMS_ADMIN + ")"
-	)
-	private Portlet _portlet;
```
Comment (MarinhoFeliphe) [inline modules/dxp/apps/portal-workflow-kaleo-forms/portal-workflow-kaleo-forms-web/src/main/java/com/liferay/portal/workflow/kaleo/forms/web/internal/asset/KaleoProcessAssetRendererFactory.java:119]: Do not remove this one, please see the reason why [here](https://github.com/liferay-workflow/liferay-portal/pull/366/files)

cc: @pedro-oliveira446 @diogomelo09 @guilhermedcamacho @gabrielwas

File: modules/dxp/apps/portal-workflow-kaleo-forms/portal-workflow-kaleo-forms-service/src/main/java/com/liferay/portal/workflow/kaleo/forms/internal/instance/lifecycle/AddDefaultKaleoProcessStructuresPortalInstanceLifecycleListener.java [hunk 2]
```diff
@@ -62,12 +60,6 @@ public void portalInstanceRegistered(Company company) throws Exception {
 	@Reference
 	private GroupLocalService _groupLocalService;
 
-	@Reference
-	private KaleoProcessLocalService _kaleoProcessLocalService;
-
-	@Reference(target = ModuleServiceLifecycle.PORTAL_INITIALIZED)
-	private ModuleServiceLifecycle _moduleServiceLifecycle;
```
Comment (MarinhoFeliphe) [inline modules/dxp/apps/portal-workflow-kaleo-forms/portal-workflow-kaleo-forms-service/src/main/java/com/liferay/portal/workflow/kaleo/forms/internal/instance/lifecycle/AddDefaultKaleoProcessStructuresPortalInstanceLifecycleListener.java:69]: Do not remove `ModuleServiceLifecycle.PORTAL_INITIALIZED` it's used to ensure the component will be activated after the portal is initialized.

Thanks!

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/java/com/liferay/portal/workflow/metrics/internal/background/task/WorkflowMetricsSLAProcessBackgroundTaskHelper.java [hunk 3]
```diff
@@ -107,21 +105,12 @@ private String _getBackgroundTaskName(
 	@Reference
 	private BackgroundTaskLocalService _backgroundTaskLocalService;
 
-	@Reference(target = ModuleServiceLifecycle.PORTLETS_INITIALIZED)
-	private ModuleServiceLifecycle _moduleServiceLifecycle;
```
Comment (MarinhoFeliphe) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/java/com/liferay/portal/workflow/metrics/internal/background/task/WorkflowMetricsSLAProcessBackgroundTaskHelper.java:111]: Do not remove ModuleServiceLifecycle.PORTAL_INITIALIZED it's used to ensure the component will be activated after the portal is initialized.

Thanks!

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/java/com/liferay/portal/workflow/metrics/internal/background/task/WorkflowMetricsSLAProcessBackgroundTaskHelper.java [hunk 4]
```diff
@@ -107,21 +105,12 @@ private String _getBackgroundTaskName(
...
 
 	@Reference
 	private WorkflowMetricsSLADefinitionLocalService
 		_workflowMetricsSLADefinitionLocalService;
-
-	@Reference(
-		target = "(background.task.executor.class.name=com.liferay.portal.workflow.metrics.internal.background.task.WorkflowMetricsSLAProcessBackgroundTaskExecutor)"
-	)
-	private BackgroundTaskExecutor
-		_workflowMetricsSLAProcessBackgroundTaskExecutor;
```
Comment (MarinhoFeliphe) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/java/com/liferay/portal/workflow/metrics/internal/background/task/WorkflowMetricsSLAProcessBackgroundTaskHelper.java:125]: Do not remove this one, it's used to ensure `WorkflowMetricsSLAProcessBackgroundTaskExecutor` is activated before the method that call the background process is called

**Comment by pedro-oliveira446** [comment]
Resent https://github.com/liferay-workflow/liferay-portal/pull/1063

**Pull request closed without merging**